src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -157,12 +157,14 @@ create_deactivate()
     # unset the variable
     for var in $changed
     do
-        val=$(grep -e "^export $var=" env.orig | cut -f 2- -d '=')
-        if test -z "$val"
+        orig=$(grep -e "^export $var=")
```

Comment:
string was overcut.. missed the variable `env.org` -> `orig=$(grep -e "^export $var=" env.orig)`

Reply:
Yeah, saw that too - I pushed before the test run completed... *blush*